plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/core/startup/QMigrationActivity.kt

```diff
@@ -63,7 +63,7 @@ class QMigrationActivity : StartupActivity.DumbAware {
                 project = project,
                 notificationActions = listOf(
                     NotificationAction.createSimple(message("aws.q.migration.action.read_more.text")) {
-                        BrowserUtil.browse(URI(Q_MARKETPLACE_URI))
+                        BrowserUtil.browse("https://plugins.jetbrains.com/plugin/24267-amazon-q")
```

Comment:
nit: How about using a constant variable to represent the link rather than directly embedding the URL?